dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops

```diff
@@ -51,4 +51,8 @@ CWWKF1705.failed.response.useraction=Fix the communication issue and try again.
 
 CWWKF1706.issue.parsing=CWWKF1706W: Issue when parsing Product Information.
 CWWKF1706.issue.parsing.explanation=The service cannot successfully parse the product information.
-CWWKF1706.issue.parsing.useraction=Fix the communication issue and try again.
\ No newline at end of file
+CWWKF1706.issue.parsing.useraction=Fix the communication issue and try again.
+
+CWWKF1707.internal.error=CWWKF1706W: An internal error occurred while checking for security vulnerabilities. The error is: {0}
```

Comment:
```suggestion
CWWKF1707.internal.error=CWWKF1706W: An internal error occurred when the system checked for security vulnerabilities. The error is: {0}
```

Reply:
not sure if "system" is the correct term here, but we should avoid elliptical clause "while checking", which can cause translation problems. Also, better state _what_ is doing the checking if possible.

Reply:
Updated to: ```An internal error occurred when the Liberty runtime checked for security vulnerabilities. The error is: {0}```